src/index.js

```diff
@@ -5,6 +5,7 @@ import StateIndicatorsService from './state-indicators.service'
 angular
   .module('angularNavStateIndicators', [
     'angularNavData',
-    'angularNavThresholds'
+    'angularNavThresholds',
+    'angularNavData.products'
```

Comment:
This last line is causing the problem "Error: dataModuleRemoteDB should be provided in the data module configuration (line 1)"

Just remove `angularNavData.products`, you don't need to add it as a dependency since `angularNavData` (added before) already includes it.